### PR TITLE
Feature/561 color sensitive red green

### PR DESCRIPTION
**Pull request by maxdiebold-erg**
## Related Issues:
* [HMW-561](https://jira.epa.gov/browse/HMW-561)

## Main Changes:
* Changed some literal color values to variables, to make it easier to change them in the future.
* Updated the color palette to be more accessible to individuals with color sensitivity.
  * I used the tool [Coloring for Colorblindness](https://davidmathlogic.com/colorblind/#%23D81B60-%231E88E5-%23FFC107-%23004D40) by David Nichols to tweak the colors, attempting to address all sensitivities without changing the colors too drastically. The previous and proposed palettes are shown below:

### [Previous Palette](https://davidmathlogic.com/colorblind/#%23CB223E-%23FFA500-%23FFFE00-%2320800C-%23007BFF-%236C95CE-%236B4195-%23C500FF)
colors-original

### [Updated Palette](https://davidmathlogic.com/colorblind/#%23D80C77-%23FFB000-%23FFFF00-%23008A01-%231A85FF-%2300CACA-%235C1A9E-%23D877FF)
colors-updated

## Steps To Test:
1. Go to http://localhost:3000/community/lake%20okeechobee/monitoring and test the appearance of the various water monitoring features, as well as polygon waterbodies.
2. Navigate to the restore tab and select "Restoration Plans" to view the color of waterbodies with no status.
3. Go to http://localhost:3000/tribe/PUEBLOOFTESUQUE to test the point waterbody features beside other point features. The color of the "Past Water Conditions" features was lightened, so this may satisfy [HMW-556](https://jira.epa.gov/browse/HMW-556).

### CODE REVIEW

**Comment by maxdiebold-erg** [comment]
@cschwinderg We may want to show this to EPA team before merging it in, but I just wanted to get it on the table.

**Review by cschwinderg — APPROVED** [review]
Looks good! Your methodology makes sense. 

I'll hold off on merging this in and we can show it to EPA on Thursday.

**Pull request merged**
